app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+require 'bcrypt'
+class User
+
+  attr_reader :password
+  attr_accessor :password_confirmation
+
+  include DataMapper::Resource
+
+  validates_confirmation_of :password
```

Comment:
Not able to get this to work as hound requests, I think it may be Rails specific I'll keep looking for a Datamapper example